waspc/data/Generator/templates/react-app/src/components/Loader.tsx

```diff
@@ -1,51 +1,12 @@
-import { styled, keyframes } from 'wasp/core/stitches.config'
+import styles from './Loader.module.css'
 
-const fullRotationKeyframes = keyframes({
-  '0%': { transform: 'rotate(0deg)' },
-  '100%': { transform: 'rotate(360deg)' },
-})
-
-// DefaultLoader is a React component that spans accross the whole screen and displays
-// a spinner in the very middle.
+// Spans a accross the whole screen and displays a spinner in the very middle.
 export function Loader() {
+  const loaderClassName = ['loader', styles.loader].join(' ')
```

Comment:
I would maybe just inline this, but do as you like.

Reply:
Btw why do you also need to set 'loader' class here? I would imagine only `styles.loader` is enough?

Reply:
In that case, shouldn't we name it sometihng like `wasp-loader`? Because this is now global CSS class if I am right -> calling it just `loader` seems problematic.

Reply:
Most people use Tailwind these days or one of the CSS-in-JS options, so I don't think there will be a lot of interference with built in classes. If it happens to happen, let's change it then.

Reply:
So you are saying that our global CSS hopefully wont cause any issues because others might not use global CSS themselves? That is ok, but we can't count on that. Also, as a library/framework, we should at least try to not affect their global space if we can. I think we should do this now, it is an easy change and is a good practice and we know it has a potential to cause problems down the road: it would be downright sloppy to leave this as it is now.

Reply:
Sure, I can add the `wasp-` prefix in a new PR.